install/0000_30_machine-api-operator_00_credentials-request.yaml

```
@@ -218,3 +218,28 @@ spec:
   secretRef:
     namespace: openshift-machine-api
     name: powervs-credentials
+---
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-machine-api-alibabacloud
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AlibabaCloudProviderSpec
+    statementEntries:
+      - action:
+          - ecs:*
```

Updating suggestion from [this comment](https://github.com/openshift/machine-api-operator/pull/926#pullrequestreview-814487051)
```suggestion
          - ecs:DeleteInstances
          - ecs:DescribeImages
          - ecs:DescribeInstances
          - ecs:DescribeSecurityGroups
          - ecs:RunInstances
          - ecs:StopInstances
          - ecs:TagResources
```

---

Updated.